charts/akeyless-api-gateway/templates/cache.yaml

```diff
@@ -60,40 +72,65 @@ spec:
               #!/usr/bin/env bash -e
               export REDISCLI_AUTH="$REDIS_PASSWORD"
               response=$(
-                  redis-cli ping
+                redis-cli {{ $tlsArgs }} ping
               )
               if [ "$response" != "PONG" ] ; then
                   echo "Unable to get keys, something is wrong"
                   echo "$response"
                   exit 1
-                fi
+              fi
           initialDelaySeconds: 5
           periodSeconds: 5
         resources:
-{{- toYaml .Values.cache.resources | nindent 12 }}
+          {{- toYaml .Values.cache.resources | nindent 12 }}
         env:
         - name: REDIS_DISABLE_COMMANDS
           value: "FLUSHDB,FLUSHALL"
         - name: REDIS_EXTRA_FLAGS
-          value: "--maxmemory {{ include "akeyless-api-gw.redisMaxmemory" . }} --maxmemory-policy allkeys-lru"
+          value: '--maxmemory {{ include "akeyless-api-gw.redisMaxmemory" . }} --maxmemory-policy allkeys-lru'
         - name: REDIS_PASSWORD
           valueFrom:
             secretKeyRef:
               name: {{ include "akeyless-api-gw.cacheSecretName" . }}
               key: cache-pass
+        - name: REDIS_TLS_ENABLED
+          value: {{ include "cache_enable_tls" . | quote -}}
+        {{- if (include "cache_enable_tls" . | eq "true") }}
+        - name: REDIS_TLS_PORT
```

Comment:
why do we need REDIS_TLS_PORT and REDIS_PORT ? shouldn't we have only one port tls or not?

Reply:
Both should be configured in the case TLS is enabled. 
The svc only exposes the TLS port